nixtract/describe-derivation.nix

```diff
@@ -34,7 +34,13 @@ in
       pname = (builtins.tryEval (targetValue.pname or false)).value or null;
       version = (builtins.tryEval (targetValue.version or "")).value;
       broken = (builtins.tryEval (targetValue.meta.broken or false)).value;
-      license = (builtins.tryEval (targetValue.meta.license.fullName or "")).value;
+      license = (builtins.tryEval (
```

Comment:
For now I'd rather we output not only the spdx id of the license but all fields, as not all licenses have a spdxId.